docs/source/bentocloud/how-tos/deploy-bentos.rst

```diff
@@ -15,46 +15,152 @@ Online Service
+-----------------------
+
+After your Bento is deployed, do the following to check the status of the deployment:
+
+1. Navigate to the `Deployments <http://cloud.bentoml.com/deployment>`_ section.
+2. Select the desired deployment. On the deployment details page, view the deployment information, such as status, events, replicas, and revisions.
+
+   .. image:: ../../_static/img/bentocloud/viewing-deployment.gif
+
+3. To update the deployment, click **Update** in the upper-right corner, update your desired field, and click **Submit**.
+
+---------------------------
+Terminating Your Deployment
+---------------------------
+
+You can temporarily stop a Bento deployment to make its endpoint inaccessible. The terminated deployment can be restarted later and all the revision records are preserved.
+
+To terminate a deployment, do the following:
+
+1. On the deployment details page, click **Terminate** in the upper-right corner.
+2. In the dialog that appears, enter the deployment's name.
+3. Click **Terminate**.
+
+You can restart the deployment to make it available again by clicking **Restore**.
+
+------------------------
+Deleting Your Deployment
+------------------------
+
+After a deployment is terminated, you can delete it. All the revision records will be deleted as well.
+
+To delete a deployment, do the following:
+
+1. On the deployment details page, click **Delete** in the upper-right corner.
+2. In the dialog that appears, enter the deployment's name.
+3. Click **Delete**.
+
+.. warning::
+
+   You can't recover a deployment after deleting it. This action is irreversible.
+
+Managing Deployments with the BentoML CLI
+=========================================
+
+The BentoML CLI is a set of tools that you can use to deploy any machine learning models as production-ready API endpoints on the cloud.
+To create and manage your Bento deployments on BentoCloud, use ``bentoml deployment`` with the corresponding options.
+
+--------------------
+Deploying Your Bento
+--------------------
+Currently, the BentoML CLI only supports creating and updating a Bento deployment by specifying a JSON file, which contains detailed configurations of the deployment, such as ``name``, ``mode``, and ``targets``.
+The JSON file follows the same syntax as the **JSON** tab when you create or update a deployment on the BentoCloud Console.
+
+Run the following command to deploy a Bento.
+
+.. code-block:: bash
+
+   bentoml deployment create --file <file_name>.json
+
+-----------------------
+Viewing Your Deployment
+-----------------------
+
+Run the following command to view all the existing deployments on BentoCloud:
+
+.. code-block:: bash
+
+   bentoml deployment list
+
+Run the following command to view the detailed information about a specific Bento deployment:
+
+.. code-block:: bash
+
+   bentoml deployment get <deployment_name>
+
+Run the following command to update a deployment.
+
+.. code-block:: bash
+
+   bentoml deployment update --file <file_name>.json
```

Comment:
Same here. Would be nice if we can make a tab with sphinx-design

Reply:
Tab added. Preview:
![Screenshot 2023-06-28 at 10 30 24](https://github.com/bentoml/BentoML/assets/65327072/d452d7eb-c5f0-4e81-aa68-5deecdeced7a)